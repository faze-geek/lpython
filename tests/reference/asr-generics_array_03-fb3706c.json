{
    "basename": "asr-generics_array_03-fb3706c",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/generics_array_03.py",
    "infile_hash": "66caaf36c724826274bb5288573e19175417a30c8470ce092e569adc",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-generics_array_03-fb3706c.stderr",
    "stderr_hash": "f903de9757cc2bd647630aa72d8f8633c503e0d7aaae81e1554ad94c",
    "returncode": 2
}