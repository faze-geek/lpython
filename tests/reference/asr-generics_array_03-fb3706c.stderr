semantic error: 'i32' object does not support item assignment
  --> tests/../integration_tests/generics_array_03.py:32:5
   |
32 |     a_int[0,0] = 400
   |     ^^^^^^^^^^^^^^^^ 
